test/checks/keyboard/focusable-disabled.js

```diff
@@ -41,6 +41,50 @@ describe('focusable-disabled', function() {
     assert.isTrue(actual);
   });
 
+  it('returns true when content made unfocusable through disabled fieldset', function() {
+    var params = checkSetup(
+      '<fieldset disabled aria-hidden="true"><input id="target" /></fieldset>'
```

Comment:
These tests are technically running against the wrong node. The evaluate function should be run against the element that has the `aria-hidden` on it. Each of these tests (except for the shadow root one) runs against an input element. This can potentially cause problems later on if something about how we get tabbableElements stops returning the node itself.